src/doc/python_scripting/functions.rst

```diff
@@ -1889,6 +1889,52 @@ return type : CLI_return_t
   DrawPlots()
 
 
+SuspendGUIUpdates
+-----------------
+
+**Synopsis:**
+
+::
+
+  SuspendGUIupdates() -> integer
+
+return type : CLI_return_t
+    The DefineVectorExpression function returns 1 on success and 0 on failure.
+
+**Description:**
+
+    SuspendGUIUpdates temporarily suspends updating the GUI after certain CLI
```

Comment:
Does our single line per sentence guidelines apply in this file?

Reply:
Yes and no. The tools have been upgraded to wrap the text that gets put into `MethodDoc.C`. So, now, yes, you can avoid having to wrap lines manually in this file. But, that did not used to be the case. It used to be that `MethodDoc.C` was a *verbatim* copy of the lines here so that it would display properly in the python interpreter.

Reply:
I was asking because we do single-sentence per line to make it easier to check diffs during code review.

Reply:
Fixed now.